packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/ImportNFTs/index.tsx

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import { useDispatch } from 'react-redux'
+import { colors } from '@blockchain-com/constellation'
+import styled from 'styled-components'
+
+import {
+  Button,
+  Image,
+  Link,
+  SkeletonCircle,
+  SkeletonRectangle,
+  Text
+} from 'blockchain-info-components'
+import CryptoAddress from 'components/CryptoAddress/CryptoAddress'
+import { Flex } from 'components/Flex'
+import FlyoutHeader from 'components/Flyout/Header'
+import QRCodeWrapper from 'components/QRCode/Wrapper'
+import { actions } from 'data'
+import { media } from 'services/styles'
+
+import { Props as OwnProps } from '..'
+
+const ImportNFTs: React.FC<Props> = (props) => {
+  const { close, defaultEthAddr } = props
+  const dispatch = useDispatch()
+
+  const CustomText = styled(Text)`
+    text-align: center;
+    width: 418px;
+    ${media.tablet`
+  width: 100%;
+  `}
+  `
+  const QRCodeContainer = styled.div`
```

Comment:
I would recommend importing the `<Flex />` and `<Padding />` components here and using them so you don't can get rid of some of these extra components. 
https://blockchain.github.io/blockchain-wallet-v4-frontend/?path=/story/structural-flex--icon-list-item-example
https://blockchain.github.io/blockchain-wallet-v4-frontend/?path=/story/structural-padding-intro--page

not a deal breaker though

Reply:
oh... you did already.